xpla/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "xpla",
+  "assets": [
+    {
+      "description": "The native staking token of XPLA.",
+      "denom_units": [
+        {
+          "denom": "axpla",
+          "exponent": 0
+        },
+        {
+          "denom": "xpla",
+          "exponent": 18
+        }
+      ],
+      "base": "axpla",
+      "name": "Xpla",
+      "display": "xpla",
+      "symbol": "XPLA",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/xpla/images/xpla.svg",
+	      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/xpla/images/xpla.png"
```

Comment:
Please either change these references to site the upper case file names (e.g., .../images/xpla.png -> .../images/XPLA.png), or else make the actual image names lower case (preferred) (e.g., XPLA.png -> xpla.png).

Reply:
Change file name of images